src/ExpoMessage.php

```diff
@@ -6,182 +6,80 @@
 
 class ExpoMessage
 {
-    /**
-     * The message title.
-     *
-     * @var string
-     */
-    protected $title;
-
-    /**
-     * The message body.
-     *
-     * @var string
-     */
-    protected $body;
-
-    /**
-     * The sound to play when the recipient receives this notification.
-     *
-     * @var string|null
-     */
-    protected $sound = 'default';
-
-    /**
-     * The number to display next to the push notification (iOS).
-     * Specify zero to clear the badge.
-     *
-     * @var int
-     */
-    protected $badge = 0;
-
-    /**
-     * The number of seconds for which the message may be kept around for redelivery if it has not been delivered yet.
-     *
-     * @var int
-     */
-    protected $ttl = 0;
-
-    /**
-     * ID of the Notification Channel through which to display this notification on Android devices.
-     *
-     * @var string
-     */
-    protected $channelId = '';
-
-    /**
-     * The json data attached to the message.
-     *
-     * @var string
-     */
-    protected $jsonData = '{}';
-
-    /**
-     * The priority of notification message for Android devices.
-     *
-     * @var string
-     */
-    protected $priority = 'default';
-
-    /**
-     * Create a message with given body.
-     *
-     * @param  string  $body
-     * @return static
-     */
-    public static function create($body = '')
+    protected string $title;
+
+    protected ?string $sound = 'default';
+
+    protected int $badge = 0;
+
+    protected int $ttl = 0;
+
+    protected string $channelId = '';
+
+    protected string $jsonData = '{}';
+
+    protected string $priority = 'default';
+
+    public static function create(string $body = ''): static
```

Comment:
let's move it to constructor as all of them has the default value (except the title, but we can make it nullable)